drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <arch/cpu.h>
+#include <init.h>
+#include <sys/printk.h>
+#include "intc_ite_it8xxx2.h"
```

Comment:
I think this include should go below the next one, to keep the ordering

Also sw comes before sys